frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -3,21 +3,233 @@ import { Scoped } from "kremling";
+          />
+          <small className="web-form-input">(Maximum $35.00)</small>
+
+          <CheckBox
+            dataKey="case.divorceOrientation"
+            label="Divorce orientation class fee"
+          />
+          <small className="web-form-input">(Maximum $30.00)</small>
+
+          <CheckBox
+            dataKey="case.vitalRecordFee"
+            label="Office of Vital Records fee"
+          />
+          <small className="web-form-input">
+            (Certification of Adoption or Certificate of Divorce; $8.00)
+          </small>
+
+          <CheckBox dataKey="case.serviceFee" label="Service Fee" />
+          <small className="web-form-input">(within Utah)</small>
+
+          <CheckBox dataKey="case.appealFee" label="Appeal fee" />
+          {data.case.appealFee && (
+            <>
+              <CheckBox label="$225 Filing fee" dataKey="case.appealFeeLarge" />
+              <CheckBox
+                label="$10 small claims appeal"
+                dataKey="case.appealFeeSmall"
+              />
+              <small className="web-form-input">(Justice Court)</small>
+            </>
+          )}
+        </Section>
+        <Section name="4. Employment">
+          <CheckBox dataKey="case.employed" label="I am employed" />
+          {data.case.employed && (
+            <>
+              <small className="web-form-input">(Choose all that apply):</small>
+              <CheckBox
+                dataKey="case.hourlyEmployee"
+                label="An hourly employee"
+              />
+              <small className="web-form-input">(Form W-2)</small>
+              <CheckBox
+                dataKey="case.salaryEmployee"
+                label="A salaried employee"
+              />
+              <small className="web-form-input">(Form W-2)</small>
+              <CheckBox dataKey="case.selfEmployed" label="Self-employeed" />
+              <small className="web-form-input">
+                (Form 1099, Form K-1, Schedule C, etc.)
+              </small>
+              <CheckBox dataKey="case.otherEmployment" label="other" />
+              {data.case.otherEmployment && (
+                <TextInput
+                  dataKey="case.otherEmployeeExplain"
+                  label="Explain"
+                />
+              )}
+            </>
+          )}
+          <CheckBox dataKey="case.unemployed" label="I am unemployed" />
+          {data.case.unemployed && (
+            <TextArea
+              dataKey="case.unemployedExplanation"
+              label="I am unemployed because"
+            />
+          )}
+        </Section>
+        <Section name="5. Dependents">
+          <p className="web-form-input">
+            The following people depend on me for support
+          </p>
+          <small className="web-form-input">
+            (Count Spouse, children or other dependents in your house hold.)
+          </small>
+          <TextInput label="Number of adults" />
```

Comment:
These are missing dataKeys. Is that on purpose? Are you planning on adding the dataKeys later?

Reply:
Yes I was trying to get a rough layout of the form. Will be adding dataKeys as needed.

Reply:
👍